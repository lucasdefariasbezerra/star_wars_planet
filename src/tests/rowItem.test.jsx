import React from 'react';
import RowItem from '../components/shared/rowItem';
import { shallow } from 'enzyme';

let wrapper;
const props = {
    columns: 1,
    imageSrc: 'image.png'
};

describe('</RowItem>', () => {
    beforeEach(() => {
        wrapper = shallow(
            <RowItem { ...props} />
        );
    });

    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });

});