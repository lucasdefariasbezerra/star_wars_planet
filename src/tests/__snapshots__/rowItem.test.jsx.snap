// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`</RowItem> should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RowItem
    columns={1}
    imageSrc="image.png"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <Image
        as="img"
        src="image.png"
        ui={true}
      />,
      "columns": 1,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "as": "img",
        "src": "image.png",
        "ui": true,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <Image
          as="img"
          src="image.png"
          ui={true}
        />,
        "columns": 1,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "as": "img",
          "src": "image.png",
          "ui": true,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
